mkdocs.yml

```diff
@@ -23,6 +23,7 @@ nav:
       - Change Log Level: 'debugging/docs/change-log-level.md'
   - Node Placement:
       - EKS Node placement: 'node-placement/docs/eks-node-placement.md'
```

Comment:
Should we add a link from the EKS Node placement page to the Fargate page?

Reply:
They both could be at the same level under Node Placement too.